### PR TITLE
Added Rspec and need approval for travis CI

### PR DESCRIPTION
**Pull request by wphoong**
1. Added the rspec gem in the gemfile

2. requesting approval for Travis CI

3. created .travis.yml file in the root directory and added configuration from the documentation

### CODE REVIEW

**Comment by roderickjoseph** [comment]
Approved 3rd party access for Travis CI

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+  - 2.2
+  - jruby
```
Comment (mscoutermarsh) [inline .travis.yml]: These are the versions of Ruby that travis will run our tests against. Is only helpful when publishing a gem.

Can remove `jruby` and change 2.2 to whatever the latest ruby is, Something like `2.4.1`

**Comment by wphoong** [comment]
I accidentally merged 'master' into the branch 'rpsec' instead of the other way around. How do I change this/revert it back? or can we ignore that merge when approving?

**Comment by mscoutermarsh** [comment]
@wphoong totally fine, causes no harm. no need to do anything, can do normal merge into master now

**Comment by wphoong** [comment]
Okay, then everything should be good and working! w00t!

**Pull request merged**